nav2_regulated_pure_pursuit_controller/include/nav2_regulated_pure_pursuit_controller/regulated_pure_pursuit_controller.hpp

```diff
@@ -244,20 +245,118 @@ class RegulatedPurePursuitController : public nav2_core::Controller
     const geometry_msgs::msg::Point & p2,
     double r);
 
+  /**
+    * @brief normalize
+    * Normalizes the angle to be -M_PI circle to +M_PI circle
+    * It takes and returns radians.
+    * @param angles Angles to normalize
+    * @return normalized angles
+    */
+  template<typename T>
+  auto normalize_angles(const T & angles)
+  {
+    auto && theta = fmod(angles + M_PI, 2.0 * M_PI);
+    if (theta < 0)
+      theta += 360;
```

Comment:
Isn't this in radians?

Reply:
yes radians, I translated a function from degree to radians but it looks like I forgot to update `theta += 360` to `theta += 2.0 * M_PI`